doc-content/shared-kie-docs/src/main/asciidoc/Workbench/SecurityManagement/business-central-security-management-provider-con.adoc

```diff
@@ -0,0 +1,11 @@
+[id='business-central-security-management-provider-con_{context}']
+
+= Security management providers
+
+In the context of security management, a realm restricts access to different application resources. Realms contain information about the users, groups, roles, and permissions. A concrete user and group management service implementation for a specific realm is called a security management provider.
+
+If the built-in security management providers do not meet the requirements of your application security realm, then you can build and register your own security management provider.
+
+NOTE: If the security management provider is not installed, the user interface for managing the security realm will not be available. After you install and configure a security management provider, the user and group management features are automatically enabled in the security management user interface.
```

Comment:
NOTE: If a security management provider is not installed, the user interface for managing the security realm is not available.